### PR TITLE
[docs](jdbc catalog) fix oracle and sap hana create examples and instructions

### PR DESCRIPTION
**Pull request by zy-kkk**

### CODE REVIEW

File: docs/lakehouse/database/oracle.md [hunk 1]
```diff
@@ -50,7 +50,11 @@ CREATE CATALOG oracle PROPERTIES (
 ```
 
 :::info remarks
```
Comment (shasevendow) [inline docs/lakehouse/database/oracle.md:52]: Please use the following syntax for admonitions:

:::info Note
:::

File: docs/lakehouse/database/sap-hana.md [hunk 2]
```diff
@@ -43,12 +43,16 @@ CREATE CATALOG saphana PROPERTIES (
...
     "user"="USERNAME",
     "password"="PASSWORD",
-    "jdbc_url" = "jdbc:sap://example.net:30015/DATABASE",
+    "jdbc_url" = "jdbc:sap://Hostname:Port/?optionalparameters",
     "driver_url" = "ngdbc-2.4.51.jar",
     "driver_class" = "com.sap.db.jdbc.Driver"
 )
 ```
 
+:::info remarks
```
Comment (shasevendow) [inline docs/lakehouse/database/sap-hana.md:52]: Please use the following syntax for admonitions:

:::info Note
:::

File: i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/database/oracle.md [hunk 3]
```diff
@@ -50,7 +50,11 @@ CREATE CATALOG oracle PROPERTIES (
 ```
 
 :::info 备注
-jdbc_url 的正确语法因 Oracle 版本和配置而异。以上示例 URL 连接到名为`orcl`的 Oracle SID。
+jdbc_url 定义要传递给 JDBC 驱动程序的连接信息和参数。
```
Comment (shasevendow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/current/lakehouse/database/oracle.md:53]: jdbc_url -> `jdbc_url`

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0/lakehouse/database/oracle.md [hunk 4]
```diff
@@ -50,7 +50,11 @@ CREATE CATALOG oracle PROPERTIES (
 ```
 
 :::info 备注
-jdbc_url 的正确语法因 Oracle 版本和配置而异。以上示例 URL 连接到名为`orcl`的 Oracle SID。
+jdbc_url 定义要传递给 JDBC 驱动程序的连接信息和参数。
```
Comment (shasevendow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.0/lakehouse/database/oracle.md:53]: jdbc_url -> `jdbc_url`

File: i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/lakehouse/database/oracle.md [hunk 5]
```diff
@@ -50,7 +50,11 @@ CREATE CATALOG oracle PROPERTIES (
 ```
 
 :::info 备注
-jdbc_url 的正确语法因 Oracle 版本和配置而异。以上示例 URL 连接到名为`orcl`的 Oracle SID。
+jdbc_url 定义要传递给 JDBC 驱动程序的连接信息和参数。
```
Comment (shasevendow) [inline i18n/zh-CN/docusaurus-plugin-content-docs/version-2.1/lakehouse/database/oracle.md:53]: jdbc_url -> `jdbc_url`

File: versioned_docs/version-2.1/lakehouse/database/oracle.md [hunk 6]
```diff
@@ -50,7 +50,11 @@ CREATE CATALOG oracle PROPERTIES (
 ```
 
 :::info remarks
-The correct syntax for jdbc_url varies depending on Oracle version and configuration. The above example URL connects to an Oracle SID named `orcl`.
+jdbc_url defines the connection information and parameters to be passed to the JDBC driver.
```
Comment (shasevendow) [inline versioned_docs/version-2.1/lakehouse/database/oracle.md:53]: jdbc_url -> `jdbc_url`

**Review by luzhijing — APPROVED** [review]
LGTM

**Review by shasevendow — APPROVED** [review]

**Pull request merged**